core/baseline-core-v1.0-psd01.md

```diff
@@ -566,195 +622,877 @@ Below we list and define the components of each layer. The detailed requirements
+
+**[CRXX4]>[OXX7]** If there is a BPI network and it chooses consensus on the execution of a transaction, it MUST use a common execution framework. 
+
+Note that if more than one execution framework were chosen, no consensus could be reached on the outcome of a transaction because the state representation is execution framework dependent e.g. Ethereum account state vs. a zero-knowledge-proof of account state.
+
+The last capability relevant for the transaction lifecycle management in the BPI processing layer that needs to be specified is the Storage capability.
+
+The key discussion on the Storage capability is full data persistency (write many, read many) versus partial data persistency (write once, read many). An enterprise data base such as MongoDB is an example of the former, and Ethereum is an example of the latter.
+
+In the case of a BPI we need to distinguish between proof, transaction and state data and its history and meta data associated to state objects etc. Given the need to maintain consistency between state data on the CCSM utilized by the BPI and state data in the BPI, the following requirement is key.
+
+**[RXX]** Proof, transaction, and state object data and their histories together with their integrity proofs MUST be stored as partially persistent data in the Storage capability.
+
+**[DXX]** All other data SHOULD be stored as fully persistent data in the Storage capability.
+
+This is a consequence of step (7) in Fig. 6.2 above.
+
+The Storage capability must also meet all requirements defined in [Section 7](#7-General-BPI-Storage-Capabilities)
 -------
+# 7 General BPI Storage Capabilities
+
+BPI storage is a key enabler to scale BPI stack that are either data-intensive or data sensitive or both.
+
+This document defines BPI data storage -- outside of a CCSM -- as the storing of information in a digital, machine-readable medium where the data stored is relevant for the proper functioning of the BPI stack.
+
+The document defines storage capability requirements in the following areas 
+* Security
+* Privacy
+* Integration
+* Data Orchestration
+* Partially Persistent Data
+* Fully Persistent Data
+
+## 7.1 BPI Storage Security
+
+As has been done throughout this document, there are BPI layer specific security requirements for layers and/or components which are listed below. It is assumed that the BPI Storage capabilities are required to fulfill all BPI security requirements in [Section 2](#2-Design-and-Architecture) 
+
+**[RXX]** Data in transit in a BPI MUST be encrypted.
+
+**[RXX]** Data at rest in a BPI MUST be encrypted.
+
+**[RXX]**	BPI Storage arranged in a network MUST support pairwise key/identity-relationships between storage nodes.
+
+This is also known as secure connection.
+
+**[RXX]**	BPI Storage MUST be compatible with widely used external authentication services or . 
```

Comment:
Copy past of other ones. Shall we remove or improve it?

Reply:
that is intentional ... because it is specific to BPI Storage